sdks/go/pkg/beam/core/runtime/exec/pardo.go

```diff
@@ -35,6 +36,7 @@ type ParDo struct {
 	Side    []ReStream
 	Out     []Node
 
+	PID       string
```

Comment:
I think this is the identifier of the ParDo, based on other parts of the code, in which case it should just be 'ID.'

Reply:
Sadly, it conflicts with the exported ID method.